README.md

```diff
@@ -48,6 +49,28 @@ and run `esy` in the project root. To install [Esy](https://esy.sh), `npm
 install -g esy` should cover most workflows. If you have NodeJS / `npm`
 available, `npx esy` is even shorter.
 
+### [Opam](https://opam.ocaml.org/)
+
+You can also install melange with `opam`. First, have a look at the `esy.json` file to see if any dependencies are expected to be at particular commits. For example, you may need to pin `reason` to a particular version:
```

Comment:
I'd remove this section. Reason should be forward compatible now and we shouldn't need to pin it anymore.